packages/components/ContextualMenu/src/SubmenuItem.types.ts

```diff
@@ -6,7 +6,7 @@ import { ContextualMenuItemProps, ContextualMenuItemTokens, ContextualMenuItemSt
 import { IconProps } from '@fluentui-react-native/icon';
 import { XmlProps } from 'react-native-svg';
 
-export const submenuItemName = 'submenuItem';
+export const submenuItemName = 'SubmenuItem';
```

Comment:
might as well camelcase the const var  as well?

Reply:
I'm not sure what the convention is. It seems the var name has been camelCase, but the string value should be UpperCamelCase because this name is special / used in the React tree where everything else is UpperCamelCase

Reply:
I think we are following proper convention here. vars are lower case, types/components are uppercase